business_logic/db/participantsCollection.js

```diff
@@ -20,6 +20,18 @@ function getParticipantData(id){
     return collection.find({ "_id": id });
 }
 
+function getParticipant(id){
+    return collection.find({ participant_twiitter_id: id});
+}
```

Comment:
Delete the function getParticipantData that is above this function

Reply:
deleted